.circleci/config.yml

```diff
@@ -4,9 +4,15 @@ jobs:
   build:
     working_directory: /go/src/github.com/resin-io/sshproxy
     docker:
-      - image: golang:1.8.3
+      - image: golang:1-alpine
```

Comment:
Keep this pinned to a specific version please, Go's dependency management is bad enough as it is!

Reply:
Also, why Alpine?

Reply:
Was thinking that if version is printed, then we still can know what version it was built with. Alpine is 1/3rd the size, and has no known vulnerabilities as opposed to the debian one, see https://hub.docker.com/r/library/golang/tags/

Reply:
> Alpine is 1/3rd the size, and has no known vulnerabilities as opposed to the debian one

Is this implication here that the Debian base image _does_ have known vulnerabilities? Surely not!

Alpine looks fine, anyway.

Reply:
FWIW I think people stress over the size of Alpine stuff way too much. It is 2018, does 300MB really make a difference to you? :)